tiledb/ml/models/sklearn.py

```diff
@@ -40,13 +46,19 @@ def save(
         if not update:
             self._create_array()
 
-        self._write_array(serialized_model=serialized_model, meta=meta)
+        if self.tiledb_uri and self.ctx:
+            self._write_array(uri=self.tiledb_uri, serialized_model=serialized_model, meta=meta)
+        else:
+            self._write_array(uri=self.uri, serialized_model=serialized_model, meta=meta)
```

Comment:
Gut feeling, there shouldn't be a need for both `tiledb_uri` and `uri`, one of them would suffice.

Reply:
`uri` accounts for the physical location on disk either local filesystem or s3, used in creation
Once created `tiledb_uri` is used to write